### DISCUSSION TITLE
Add Create Button on Create screen

### DISCUSSION THREAD

**Discussion by sfarrowGR** [keystonejs/keystone#7619] [Feature requests]
Hi,

It is fairly common on CMS platforms that when you are editing a item/post you can save and also to have a button to add another item to that group.  Much more user friendly, than having to save, click back on the left menu to go to the list of posts and then click new again.

**Comment by Achisingh** [comment]
@sfarrowGR thanks for the feedback. We will include your suggestion for review in future Admin UI improvements.
